### ISSUE TITLE
GUI doesn't launch

### ISSUE THREAD

**Issue by gmcoding74** [sahib/rmlint#673]
Installed rmlint today on Linux Mint 22 Wilma via the LM Repos.

Ran:

sudo rmlint --gui

Received:
 File "/tmp/.shredder-bootstrap.py.QQFJV2", line 23
    
SyntaxError: source code cannot contain null bytes

Other info:

rmlint --version
version 2.9.0 compiled: Apr 15 2021 at [21:03:37] "Odd Olm" (rev 2.3)
compiled with: +mounts +nonstripped +fiemap +sha512 +bigfiles +intl +replay +xattr +btrfs-support

python3 --version
Python 3.12.3

**Comment by rsbrux** [comment]
Similar behavior under Ubuntu Studio 24.04 LTS.  Installed via KDE Discover. On "Launch", icon appears briefly in Task Manager, then disappears.  GUI never appears.  This may not be the correct place to report this, since rmlint-gui may be a different project which just has rmlint as a dependency.  Any guidance as to the correct approach would be welcome.

**Comment by RayOei** [comment]
> This may not be the correct place to report this

No, you are fine. It fails indeed. As a prospective contributor I have taken a quick peek and suspect I know what the issue is. Can't say if this will be fixed in an official release any time soon, though.

**Comment by RayOei** [comment]
It seems this #628 tried to resolve this issue and was released in v2.10.2 but the problem has clearly not been resolved. I thought it was a null issue but at this point I ran into the problem that the import is not available which may indicate that the install doesn't install what is needed?

**Comment by RayOei** [comment]
The original issue was resolved in #628.
Other observed installation issues should be addressed by #710 and will be included in v2.10.3
